docs/protocol/network/data-structures/transaction.md

```diff
@@ -85,59 +120,45 @@ commit_value = (data_request_value / witnesses) - commit_fee
-- Commit claim: provide a signed reveal message matching a previous `commitment` (hash) and that matches with the previous advertised public key.
-- Reveal claim: provide a data request consensus result which is valid according to the consensus script and previous reveals outputs.
-
-### Value transfer claim
-
-Value transfer claims prove ownership of a certain private key by providing a signature of the identifier of the transaction produced with such key and the serialization of the matching public key.
+### Consensus outputs
 
-| Field        |  Type  | Description                                     |
-| ------------ | :----: | ----------------------------------------------- |
-| `signature`  | `[u8]` | Signed hash of the transaction                  |
-| `public_key` | `[u8]` | Public Key of the P2PKH outpoint to be consumed |
+The consensus output is included by the **block miner** and it defines the data request result after the consensus clause. Only one **consensus output** is required per data request.
 
-### Data request claim (**commit stage**)
+| Field    |  Type  | Description                                                                                               |
+| -------- | :----: | --------------------------------------------------------------------------------------------------------- |
+| `result` | `[u8]` | Data request consensus result after using all data request reveal values of the previous **reveal stage** |
+| `pkh`    | `[u8]` | Slice of public key hash (20 bytes) of the data request creator                                           |
+| `value`  | `u64`  | Remaining transaction value that has not been used as reward or fee of the data request                   |
 
-A **data request** output has to be consumed/used by a number of **witnesses**, i.e. there will be as many claims as **witnesses** have been defined. For a committer to be able to pledge a share of the reward from the data request, they must provide a _Proof of Eligibility_ (PoE) that proves their eligibility as witnesses for such data request in the current epoch. In addition, for everyone in the network to be able to verify such proof, they must include the public key that matches the private key that produced the PoE.
+The `value` of the consensus output is the remaining value after distributing all rewards and fees among witnesses and miners respectively:
 
-| Field        |  Type  | Description                           |
-| ------------ | :----: | ------------------------------------- |
-| `poe`        | `[u8]` | Proof of eligibility                  |
-| `public_key` | `[u8]` | Public Key used for computing the PoE |
-
-### Commit claim (**reveal stage**)
+```math
+reveal_value = data_request_value - committers * commit_fee - revealers * (reveal_fee + tally_fee + reward)
+```
 
-A commit claim is used to prove that the witness has a valid `reveal`, i.e. the signed data request result and nonce match the previous commitment.
+## Claims
```

Comment:
Same as before, I would just call them signatures. These they can serve as claims (in the cases of Value Transfer Outputs). I feel the name is confusing